src/test/scala/com/databricks/spark/csv/util/InferSchemaSuite.scala

```diff
@@ -2,8 +2,31 @@ package com.databricks.spark.csv.util
 
 import org.apache.spark.sql.types._
 import org.scalatest.FunSuite
+import org.scalatest.BeforeAndAfterAll
+import org.apache.spark.SparkContext
+import org.apache.spark.sql.SQLContext
+import com.databricks.spark.csv.CsvParser
+import com.databricks.spark.csv.CsvRelation
 
-class InferSchemaSuite extends FunSuite {
+class InferSchemaSuite extends FunSuite with BeforeAndAfterAll {
+
+  private val simpleDatasetFile = "src/test/resources/simple.csv"
+  private val utf8Charset = "utf-8"
+  private var sqlContext: SQLContext = _
+
+  override def beforeAll(): Unit =
```

Comment:
These can be removed once you move your end-to-end test to `CsvSuite.scala`. This suite a unit test and does not need to create a SQLContext.